scripts/exregional_run_prepstart.sh

```diff
@@ -492,7 +481,7 @@ if [ ${HH} -eq ${SST_update_hour} ] && [ ${cycle_type} == "prod" ] ; then
       cp ${SST_ROOT}/nsst.$YYYYMMDDm1/rtgssthr_grb_0.083.grib2 latest.SST
    else
      echo "${SST_ROOT} data does not exist!!"
-     echo "ERROR: No SST update at ${HH}!!!!"
+     err_exit "No SST update at ${HH}!!!!"
```

Comment:
If imssnow or SST data is not available, do we want the model to exit with a fatal error?  I am wondering if it would be better to have a warning message instead, but to continue running, which is what the HRRR does.

Reply:
Got it! I'll change it to WARNING.

Reply:
done